scripts/utils.js

```diff
@@ -22,7 +22,7 @@ export const [setLibs, getLibs] = (() => {
         if (!(hostname.includes('.hlx.') || hostname.includes('local'))) return prodLibs;
         const branch = new URLSearchParams(search).get('milolibs') || 'main';
         if (branch === 'local') return 'http://localhost:6456/libs';
-        return branch.includes('--') ? `https://${branch}.hlx.live/libs` : `https://${branch}--milo--adobecom.hlx.live/libs`;
+        return branch.includes('--') ? `https://${branch}--milo--adobecom.hlx.page/libs` : `https://${branch}--milo--adobecom.hlx.live/libs`;
```

Comment:
Nice! I think we have similar logic on bacom, so we might create a PR there too.

Reply:
FWIW: the original behavior is the expected behavior.

Reply:
Thanks for taking a look @auniverseaway but https://main.hlx.live/libs gives 404 is that intended?